### PR TITLE
These are the changes from the google roadshow lab coming from steve

### PR DESCRIPTION
**Pull request by jorgemoralespou**

### CODE REVIEW

**Comment by siamaksade** [comment]
+1

@thoraxe what do you think?

File: modules/codechanges.adoc [hunk 1]
```diff
@@ -55,7 +55,7 @@ Username: {{GITLAB_USER}}
 Password: {{GITLAB_PASSWORD}}
 ====
 
-Click the Settings link on the top right of the screen, and then click "Webhooks":
+Click the Settings link on the top right of the screen, and then click "Integrations":
```
Comment (thoraxe) [inline modules/codechanges.adoc]: Is this a change in Gitlab? When I did this the other day it still said "Webhooks".

Comment (jorgemoralespou) [inline modules/codechanges.adoc]: We've instale latest gitlab and it seems this has changed. 8.16 which by the way installs nicer as requires just anyuid for the serviceaccount for the gitlab pod and not for postgres or redis. 
cc/ @thesteve0 which wrote this part

Comment (thoraxe) [inline modules/codechanges.adoc]: I can try to update the workshop provisioner to use these. Which template are you using?

I guess just merge it -- we can talk around it if it's a problem.

**Comment by jorgemoralespou** [comment]
This is the official template, and I try always to use that one and report
them problems:

https://gitlab.com/gitlab-org/omnibus-gitlab/blob/master/docker/openshift-template.json

**Comment by thoraxe** [comment]
I try not to grab latest all the time, but I will try to update to that one and see how it works. 

I guess this PR is fine to merge.

**Pull request merged**